Controller/Controller.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Sonata package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\CoreBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller as BaseController;
+use Symfony\Component\HttpFoundation\Request;
+
+abstract class Controller extends BaseController
```

Comment:
Abstract classes should be prefixed with `Abstract`, see [Symfony naming conventions](http://symfony.com/doc/master/contributing/code/standards.html#naming-conventions).